rasa/nlu/tokenizers/whitespace_tokenizer.py

```diff
@@ -24,6 +25,27 @@ def __init__(self, component_config: Dict[Text, Any] = None) -> None:
 
         self.case_sensitive = self.component_config["case_sensitive"]
 
+        self.emoji_pattern = self.get_emoji_regex()
+
+    @staticmethod
+    def get_emoji_regex():
```

Comment:
```suggestion
    def get_emoji_regex() -> re.Pattern:
```

Reply:
I tried this and pytests are failing with `module re has no attribute Pattern`. Looks like a reported issue for python 3.6 here - https://www.reddit.com/r/inventwithpython/comments/exypps/chapter_8_error_module_re_has_no_attribute_pattern/
Will leave it out for now.